### ISSUE TITLE
Variable declared after use generate NaN

### ISSUE THREAD

**Issue by gik000** [creotiv/jquery-photowall#27]
Hi creotiv,

I found what I think is a bug, on line 209 of jquery.photowall.js...
var ln = $("<div class='pw-line' style='width:"+(total_width+num_photos*PhotoWall.options.padding*2)+"'></div>")
                     .appendTo(PhotoWall._el);
            var num_photos = line.length;       

as subject of this issue states,
the variable "num_photos" is used before it is declared and filled.

This error generates problem on MAC - chrome/safari ...
(for a strange reason Windows systems are not affected about it)

I discovered it after I've implemented a modifications to make library works with ajax.

I look forward your answer,

thanks for this nice work.

Giancarlo Diana
